packages/scss/src/components/_main.scss

```diff
@@ -1,33 +1,25 @@
-.main {
-	padding-left: _theme("component.navSide.width");
-}
-
 .main-content {
 	padding: 4rem 0;
 }
 
-.main-filters {
-	margin-bottom: 3rem;
+.navSide + .main-content {
```

Comment:
We might need to use ".navSide ~ .main-content" instead of ".navSide + .main-content" to cope with the case of div being put between the two.